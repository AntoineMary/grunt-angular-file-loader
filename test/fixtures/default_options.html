<!doctype html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <title>Document</title>
</head>
<body>
<!-- angular -->
<script src="another.js" type="text/javascript"></script>
<script src="yet-another.js" type="text/javascript"></script>
<script src="no-deps.js" type="text/javascript"></script>
<script src="module.js" type="text/javascript"></script>
<script src="module-controller.js" type="text/javascript"></script>
<script src="dep-on-non-declared.js" type="text/javascript"></script>
<script src="circular3.js" type="text/javascript"></script>
<script src="circular2.js" type="text/javascript"></script>
<script src="circular.js" type="text/javascript"></script>
<script src="another-factory.js" type="text/javascript"></script>
<!-- endangular -->

</body>
</html>