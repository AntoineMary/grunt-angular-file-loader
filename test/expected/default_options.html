<!doctype html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <title>Document</title>
</head>
<body>
<!-- angular -->
<script src="test/fixtures/another.js" type="text/javascript"></script>
<script src="test/fixtures/yet-another.js" type="text/javascript"></script>
<script src="test/fixtures/no-deps.js" type="text/javascript"></script>
<script src="test/fixtures/module.js" type="text/javascript"></script>
<script src="test/fixtures/module-controller.js" type="text/javascript"></script>
<script src="test/fixtures/dep-on-non-declared.js" type="text/javascript"></script>
<script src="test/fixtures/circular3.js" type="text/javascript"></script>
<script src="test/fixtures/circular2.js" type="text/javascript"></script>
<script src="test/fixtures/circular.js" type="text/javascript"></script>
<script src="test/fixtures/another-factory.js" type="text/javascript"></script>
<!-- endangular -->

</body>
</html>